### DISCUSSION TITLE
Does not work with V380E21_PTZ_ETH_V3.2.3.3_20190513

### DISCUSSION THREAD

**Discussion by pcmester** [Arkady23/V380-Web#2] [General]
Hi!

I tried as you suggested in the V380 firmware patching github issue, but it does not work. Nothing happens.

AppV380E21_PTZ_ETH_V3.2.3.3_20190513
HWV380E21_WF3_PTZ_ETH_20160217

**Comment by Arkady23** [comment]
Because you have a different model. I need to know the HwName of your camera. Take a screenshot of the screen where the message about the release of the new firmware appears. Or listen to the traffic when you click on the "firmware version" item in the phone app.

Do you have any saved patch?
This code must be present there. If there is, attach it to the message. I will see and make an installer for your model.

**Comment by Arkady23** [comment]
I found the hwname of your camera. Take the installer **httpd_V380E3_C_v20210606.zip**.

**Reply by pcmester** [reply]
Thank you! Unfortunately, it doesn't work still. Nothing happens.

**Reply by Arkady23** [reply]
> I found the hwname of your camera. Take the installer **httpd_V380E3_C_v20210606.zip**.

Sorry, I made a mistake, your patch is next to this, i.e. **httpd_V380E3_C3_v20210606.zip**

**Comment by Arkady23** [comment]
There is another option to find out your Hwname — it is to request support at v380technical@gmail.com, so that they send some patch for your camera. Then you can see the Hwname in this patch. Write to them that the camera is not working well and you would like to get a patch to restore it.

**Comment by Arkady23** [comment]
And more. Sometimes the WiFi is low or you enter the wrong IP address of the camera. Check if the cgi-bin folder appears on the sd- card? If the cgi-bin folder appeared, it means that the patch worked, you chose the correct patch and the correct Hwname. The web-server is working! If there is no cgi-bin folder on the sd-card, then you have a different Hwname and this patch does not suit you.

**Reply by pcmester** [reply]
Thanks, but it still does not work. The HWname seems correct, but here is the firmware screen:
![image](https://user-images.githubusercontent.com/41517442/121708548-fa3ccc80-cad7-11eb-8678-e37046d593cb.png)
The SD card only contains a recording folder

**Comment by Arkady23** [comment]
You need to find out exactly the hwname or request a patch.

**Reply by pcmester** [reply]
Here is the patch: https://ufile.io/a101zr2y
Apparently, it is v380_E3_C3

**Reply by Arkady23** [reply]
Very good. **Is this a fresh patch?** I can get his number. For some cameras, it is not enough to know the hwname. You also need to know the version number. I opened this patch, it has the version number 3020606. I will definitely make you a pat based on it, but I suggest waiting for one day. Because now I am making a fix in the web server. I think it will be ready tomorrow and I will turn it on for your camera.

If it's a fresh patch. Did you install it? Do you want to install it? I can assign the web server a version 1 less than this patch, I can assign it 1 more, or exactly 3020606. If you do 3020607, then you will not be able to install this patch.

**Reply by pcmester** [reply]
Yes, it is a fresh patch. I haven't installed it yet, but I'd like to.

**Reply by Arkady23** [reply]
Then you will install it. And next time, a web-server.

**Reply by Arkady23** [reply]
Good luck!!!
https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E3_C3_3020607_v20210620.zip

**Reply by pcmester** [reply]
Thank you, it worked flawlessly!

**Reply by pcmester** [reply]
Do you know, how to play audio? The audio player on the system crashes the camera, and it reboots if I try to play something

**Reply by Arkady23** [reply]
Don't know yet.
